config/globalaccelerator/config.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package globalaccelerator
+
+import (
+	"github.com/crossplane/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-aws/config/common"
+)
+
+// Configure adds configurations for ebs group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_globalaccelerator_accelerator", func(r *config.Resource) {
+		r.Version = common.VersionV1Alpha2
+		r.ExternalName = config.NameAsIdentifier
```

Comment:
[This config](https://github.com/crossplane/terrajet/blob/main/pkg/config/defaults.go#L36) uses external name annotation as both TF ID and `name` parameter in TF config but from what I can tell, TF ID is a full ARN that doesn't include the name at all. So, `name` parameter is essentially just a label of some sort. I'd suggest using `config.IdentifierFromProvider` for this one, too.

Reply:
Thanks @muvaf, I've updated the PR with the feedback